recipe/meta.yaml

```diff
@@ -15,6 +15,9 @@ source:
     - patches/0004-fix-python-executable.patch
     - patches/0005-fix-installation-on-windows.patch
     - patches/0006-use-find-python.patch
+    # upstream seems to have disabled NumPy 2.0 for CI reasons only
+    # see https://github.com/Farama-Foundation/Arcade-Learning-Environment/pull/533
+    - patches/0007-allow-numpy-20.patch
```

Comment:
@pseudo-rnd-thoughts, can you confirm this impression? There doesn't seem to be much discussion/context for https://github.com/Farama-Foundation/Arcade-Learning-Environment/pull/533/commits/cde1f7d8e97dedf69d98d21550353ace421f72f0